fnal_column_analysis_tools/lookup_tools/jersf_lookup.py

```diff
@@ -5,17 +5,18 @@
 from copy import deepcopy
 import numba
 
-from numpy import sqrt,log
+from numpy import sqrt,log,exp,abs
```

Comment:
Why not use the namespace? `np.x`

Reply:
less faffing about in eval and string manipulation, but yeah it's not as clean.